.github/CODEOWNERS

```diff
@@ -32,7 +32,7 @@
 
 # Service teams
 # PRLabel: %App Configuration
-/sdk/appconfiguration/ @HarshaNalluru
+/sdk/appconfiguration/ @sima-zhu @HarshaNalluru
```

Comment:
I've started adding comments to these kinds of changes like 

```
# TODO: Remove added primary alias before merge
```

that prevent me from accidentally merging something when I'm validating a scenario like this. Then a quick ctrl+f over the PR will let me know if I'm leaving behind artifacts.